package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Arizona JS",
+  "scripts": {
+    "morphdom-cp": "cp node_modules/morphdom/dist/morphdom-esm.js priv/static/assets/js",
+    "morphdom-rm-export": "sed -i'.bak' -e \"s/export default morphdom;/self.morphdom=morphdom;/g\" priv/static/assets/js/morphdom-esm.js && rm -f priv/static/assets/js/morphdom-esm.js.bak",
+    "morphdom-wrap": "echo \"(function() {\n$(cat priv/static/assets/js/morphdom-esm.js)\n}());\" > priv/static/assets/js/morphdom-esm.js",
+    "morphdom-uglify": "uglifyjs priv/static/assets/js/morphdom-esm.js > priv/static/assets/js/morphdom.min.js",
+    "build": "npm install && npm run morphdom-cp && npm run morphdom-rm-export && npm run morphdom-wrap && npm run morphdom-uglify",
+    "prettier": "prettier priv/static/assets/js/arizona*.js --write && prettier eslint.config.mjs --write",
+    "eslint": "eslint priv/static/assets/js/arizona*.js && eslint eslint.config.mjs",
+    "ci": "npm install && npm run build && npm run prettier && npm run eslint"
+  },
+  "devDependencies": {
+    "@eslint/js": "9.5.0",
+    "eslint": "9.5.0",
+    "globals": "15.5.0",
+    "prettier": "2.8.7",
+    "uglify-js": "3.18.0"
+  },
+  "dependencies": {
+    "morphdom": "2.7.3"
```

Comment:
Some of these are not "latest version" since they'll fit in the Renovate pipeline (that we haven't tested for JS yet).